myproject/base/models.py

```diff
@@ -2,11 +2,21 @@
 
 # Create your models here.
 
+class Lists(models.Model):
+    user_id = models.IntegerField()
+    list_id = models.AutoField(primary_key=True)
+    list_name = models.CharField(max_length=32)
+    item_list = models.TextField(null=True, blank=True)
```

Comment:
u should add a `#TODO: this is temporary` on the item_list, since we are supposed to use something like a SQL join between a List and a future Item(models.Model) class. (there is only 1 entry for a list, but on the item the db table would be
list 1      item 1
list 1      item 2
list 2      item 1
list 2      item 2
list 3      item 3
and so on)

Reply:
I set up the TODO comment about list_id